src/not_found_handler.cc

```diff
@@ -0,0 +1,15 @@
+#include "not_found_handler.h"
+
+RequestHandler::Status NotFoundHandler::Init(const std::string& uri_prefix, const NginxConfig& config) {
+    // There should be nothing in the config block. Ignore any configs
+    return RequestHandler::Status::FILE_NOT_FOUND;
```

Comment:
Should we return Status OK? Since technically the request handler is ok. The response code though is not found?